### PR TITLE
HubSpot - add crud methods for Email Templates

### PR DESCRIPTION
**Pull request by MichaelRyanWebber**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @friggframework/api-module-hubspot@0.8.31-canary.216.26ae047.0
  # or 
  yarn add @friggframework/api-module-hubspot@0.8.31-canary.216.26ae047.0
  ```
</details>

### CODE REVIEW

**Review by leofmds — APPROVED** [review]
👍🏻

File: api-module-library/hubspot/api.js [hunk 1]
```diff
@@ -822,6 +825,58 @@ class Api extends OAuth2Requester {
         return this._post(options);
     }
 
+    // ***********************   Email Templates   **************************
+
+    async getEmailTemplates(query=''){
```
Comment (seanspeaks) [inline api-module-library/hubspot/api.js:830]: @MichaelRyanWebber Does this return an array?

Is now the time for us to change HubSpot methods to reflect the convention of `list` for array returns? (`listEmailTemplates`)

Or is that a major version release/something we should be more thoughtful of for backwards compatibility?

**Review by seanspeaks — APPROVED** [review]
Just one question/comment, but approved generally (thanks for the tests and details 👍 )

File: api-module-library/hubspot/api.js (hunk 1 above)
Comment (MichaelRyanWebber) [reply api-module-library/hubspot/api.js:830]: It returns an object with an array (called `objects`)
I don't actually know what the conventions are, fully. I think I had been erring towards following the API's naming convention, but I like using our own, and saying list seems great.
https://linear.app/lefthook/issue/LEF-651/update-hubspot-api-module-method-names
I'm fine with doing it now/soon, but would rather leave it out of this PR

**Comment by seanspeaks** [comment]
:rocket: PR was released in [`@friggframework/api-module-hubspot@0.8.31`](https://github.com/friggframework/frigg/releases/tag/%40friggframework/api-module-hubspot%400.8.31) :rocket:

**Pull request merged**